public/content/en/basics.md

```diff
@@ -305,12 +324,30 @@ the parent's scope
         }
         ...
 
+A special feature of D is the UFCS (Uniform Function Call Syntax (UFCS)).
```

Comment:
There is already a section about UFCS in the "D's Gems" section and I would like to have it there to not introduce too much language features in the basics chapter. That means that we shouldn't use UFCS in the code examples until it has been introduced.